<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>psa_radio_window</class>
 <widget class="QWidget" name="psa_radio_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="radio_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="psa-resources.qrc">:/backgrounds/backgrounds/bg.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="station_name">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>441</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DSEG14 Classic</family>
     <pointsize>45</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>radio off</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="curr_band">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>240</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PSA Groupe HMI Sans CS</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color: rgba(63, 63, 63, 120);
	color: rgb(255, 255, 255);
border: none;
border-radius: 11px;
}
</string>
   </property>
   <property name="text">
    <string>AM 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="presets_window">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>270</y>
     <width>340</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dots All For Now JL</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
background-color: rgba(63, 63, 63, 120);
border: none;
border-radius: 20px;
}
</string>
   </property>
   <property name="title">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="preset_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>76</y>
      <width>165</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dots All For Now JL</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QLabel{
	color: rgb(255, 255, 255);
	border: 2px dotted white;
	border-radius: 10px;
}*/

QLabel{
	margin: 5px;
	color: rgb(255, 255, 255);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="preset_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>76</y>
      <width>165</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dots All For Now JL</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QLabel{
	color: rgb(255, 255, 255);
	border: 2px dotted white;
	border-radius: 10px;
}*/

QLabel{
	margin: 5px;
	color: rgb(255, 255, 255);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="preset_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>38</y>
      <width>165</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dots All For Now JL</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QLabel{
	color: rgb(255, 255, 255);
	border: 2px dotted white;
	border-radius: 10px;
}*/

QLabel{
	margin: 5px;
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="preset_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>165</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dots All For Now JL</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QLabel{
	color: rgb(255, 255, 255);
	border: 2px dotted white;
	border-radius: 10px;
}*/

QLabel{
	margin: 5px;
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="preset_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>165</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dots All For Now JL</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QLabel{
	color: rgb(255, 255, 255);
	border: 2px dotted white;
	border-radius: 10px;
}*/

QLabel{
	margin: 5px;
	color: rgb(255, 255, 255);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="preset_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>38</y>
      <width>165</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dots All For Now JL</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QLabel{
	color: rgb(255, 255, 255);
	border: 2px dotted white;
	border-radius: 10px;
}*/

QLabel{
	margin: 5px;
	color: rgb(255, 255, 255);
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="radio_freq">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>205</y>
     <width>170</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DSEG7 Classic</family>
     <pointsize>28</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color: rgba(63, 63, 63, 120);
	color: rgb(255, 255, 255);
border: none;
border-radius: 22px;
}
</string>
   </property>
   <property name="text">
    <string>091.20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="big_date_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>550</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PSA Groupe HMI Sans CS</family>
     <pointsize>50</pointsize>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="big_clock_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>550</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PSA Groupe HMI Sans CS</family>
     <pointsize>55</pointsize>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="cd_icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="psa-resources.qrc">:/symbols/icons/disc.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cd_tracks">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PSA Groupe HMI Sans CS</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="signal_icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="psa-resources.qrc">:/symbols/icons/signal-3.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="psa-resources.qrc"/>
 </resources>
 <connections/>
</ui>
